src/custom/utils/trade.ts

```diff
@@ -95,18 +89,69 @@ export async function signAndPostOrder(params: PostOrderParams): Promise<AddUnse
   const summary = _getSummary(params)
   const receiver = recipient
 
-  const unsignedOrder: UnsignedOrder = {
-    sellToken: sellToken.address,
-    buyToken: buyToken.address,
-    sellAmount,
-    buyAmount,
-    validTo,
-    appData: appDataHash,
-    feeAmount: feeAmount?.quotient.toString() || '0',
-    kind,
-    receiver,
-    partiallyFillable: false, // Always fill or kill
+  return {
+    summary,
+    quoteId,
+    order: {
+      sellToken: sellTokenAddress,
+      buyToken: buyToken.address,
+      sellAmount,
+      buyAmount,
+      validTo,
+      appData: appDataHash,
+      feeAmount: feeAmount?.quotient.toString() || '0',
+      kind,
+      receiver,
+      partiallyFillable: false, // Always fill or kill
+    },
   }
+}
+
+export function mapUnsignedOrderToOrder(
```

Comment:
Would be great to move a new code to `cow-react` in case when it's not an override